src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -357,6 +361,23 @@ public static void setFlexyName() {
     flexyName = res;
   }
 
+  /** Read the Flexy serial number and return it. */

```

Comment:
Javadoc doesnt make sense

Reply:
Java docs are usually the last thing on my mind. Fixed